### PR TITLE
Moving grafana(1), inspector2(1), ivs(2), ssm(2), transcribe(3), transfer(1), vpc_network(1) resources to v1beta1 version

### LINKED ISSUE #495: Moving grafana(1), inspector2(1), ivs(2), ssm(2),  transcribe(3), transfer(1), vpc_network(1) resources to v1beta1 version
* `grafana`
  * [x] [aws_grafana_workspace_api_key](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/grafana_workspace_api_key)

* `inspector2`
  * [x] [aws_inspector2_enabler](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/inspector2_enabler)

* `ivs`
  * [x] [aws_ivs_channel](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ivs_channel)
  * [x] [aws_ivs_recording_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ivs_recording_configuration)

* `ssm`
  * [x] [aws_ssm_default_patch_baseline](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_default_patch_baseline)
  * [x] [aws_ssm_service_setting](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_service_setting)

* `transcribe`
  * [x] [aws_transcribe_language_model](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_language_model)
  * [x] [aws_transcribe_vocabulary](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary)
  * [x] [aws_transcribe_vocabulary_filter](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary_filter)

* `transfer`
  * [x] [aws_transfer_tag](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_tag)

* `vpc_network`
  * [x] [aws_vpc_network_performance_metric_subscription](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_network_performance_metric_subscription)

### PR DESCRIPTION
**Pull request by steperchuk**
### Description of your changes
* `grafana`
  * [x] [aws_grafana_workspace_api_key](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/grafana_workspace_api_key)

* `inspector2`
  * [x] [aws_inspector2_enabler](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/inspector2_enabler)

* `ivs`
  * [x] [aws_ivs_channel](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ivs_channel)
  * [x] [aws_ivs_recording_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ivs_recording_configuration)

* `ssm`
  * [x] [aws_ssm_default_patch_baseline](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_default_patch_baseline)
  * [x] [aws_ssm_service_setting](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_service_setting)

* `transcribe`
  * [x] [aws_transcribe_language_model](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_language_model)
  * [x] [aws_transcribe_vocabulary](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary)
  * [x] [aws_transcribe_vocabulary_filter](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary_filter)

* `transfer`
  * [x] [aws_transfer_tag](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_tag)

* `vpc_network`
  * [x] [aws_vpc_network_performance_metric_subscription](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_network_performance_metric_subscription)

Fixes #495

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
Manually
aws_grafana_workspace_api_key - depends on resource with manual intervention
aws_grafana_workspace_api_key

aws_transcribe_language_model - require file to be uploaded manually to s3
aws_transcribe_language_model

aws_transcribe_vocabulary -  require file to be uploaded manually to s3
aws_transcribe_vocabulary

Manually and Uptest
[examples/inspector2/enabler.yaml](https://github.com/upbound/provider-aws/actions/runs/4243073348)
[examples/ivs/channel.yaml](https://github.com/upbound/provider-aws/actions/runs/4243074284)
[examples/ivs/recordingconfiguration.yaml](https://github.com/upbound/provider-aws/actions/runs/4243075418)
[examples/ssm/defaultpatchbaseline.yaml](https://github.com/upbound/provider-aws/actions/runs/4243077647)
[examples/transcribe/vocabularyfilter.yaml](https://github.com/upbound/provider-aws/actions/runs/4243079932)
[examples/transfer/tag.yaml](https://github.com/upbound/provider-aws/actions/runs/4243080893)
[examples/vpc/networkperformancemetricsubscription.yaml](https://github.com/upbound/provider-aws/actions/runs/4243082527)

### CODE REVIEW

**Comment by steperchuk** [comment]
/test-examples="examples/inspector2/enabler.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/ivs/channel.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/ivs/recordingconfiguration.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/ssm/defaultpatchbaseline.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/ssm/servicesetting.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/transcribe/vocabularyfilter.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/transfer/tag.yaml"

**Comment by steperchuk** [comment]
/test-examples="examples/vpc/networkperformancemetricsubscription.yaml"

File: examples/transcribe/vocabulary.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      tag2: value3
+    vocabularyFileUri: s3://test-channel-bucket/transcribe/test1.txt
+
+---
+
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Role
+metadata:
+  annotations:
+    meta.upbound.io/example-id: transcribe/v1beta1/medicalvocabulary
```
Comment (dverveiko) [inline examples/transcribe/vocabulary.yaml:25]:
```suggestion
    meta.upbound.io/example-id: transcribe/v1beta1/vocabulary
```

File: examples/transcribe/vocabulary.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      }
+
+---
+
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    crossplane.io/external-name: test-channel-bucket
+    meta.upbound.io/example-id: transcribe/v1beta1/medicalvocabulary
```
Comment (dverveiko) [inline examples/transcribe/vocabulary.yaml:53]:
```suggestion
    meta.upbound.io/example-id: transcribe/v1beta1/vocabulary
```

File: examples/transcribe/vocabulary.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    forceDestroy: true
+    region: us-east-1
+
+---
+
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Policy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: transcribe/v1beta1/medicalvocabulary
```
Comment (dverveiko) [inline examples/transcribe/vocabulary.yaml:68]:
```suggestion
    meta.upbound.io/example-id: transcribe/v1beta1/vocabulary
```

File: examples/transcribe/vocabulary.yaml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        ]
+      }
+
+---
+
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: RolePolicyAttachment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: transcribe/v1beta1/medicalvocabulary
```
Comment (dverveiko) [inline examples/transcribe/vocabulary.yaml:97]:
```suggestion
    meta.upbound.io/example-id: transcribe/v1beta1/vocabulary
```

File: examples/transcribe/vocabulary.yaml [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    roleRef:
+      name: test-role
+
+---
+
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: Object
+metadata:
+  annotations:
+    meta.upbound.io/example-id: transcribe/v1beta1/medicalvocabulary
```
Comment (dverveiko) [inline examples/transcribe/vocabulary.yaml:114]:
```suggestion
    meta.upbound.io/example-id: transcribe/v1beta1/vocabulary
```

**Review by sergenyalcin — APPROVED** [review]
Thanks @steperchuk LGTM!

**Pull request merged**